ExampleMod/Localization/en-US.hjson

```diff
@@ -1308,8 +1425,58 @@ Mods: {
 			MinionBossEmote.Command: exminionboss
 		}
 
+		# This is the command names for mod emotes
+		# It is suggested to use a prefix like ex- for all emotes of your mod to avoid name collisions
+		# If there is a collision, one will just override the other
 		NotPresentDuringWorldGenMessage: ExampleMod was not used to generate this world, your experience with the content of ExampleMod will be degraded
-		Commands.ShowFullscreenUICommand.Description: Show the example fullscreen UI
+
+		Commands: {
+			ExampleCoinsCommand.Description: Show the coin rate UI
+
+			ExampleItemCommand: {
+				Usage:
+					'''
+					/item <type|name> [stack]
+					type — ItemID of item.
+					name — name of Item in current localization.
+					Replace spaces in item name with underscores.
+					'''
+				Description: Spawn an item by name or by typeId
+				Error_0: At least one argument was expected.
+				Error_1: Unknown item — Must be valid name or 0 < type < {0}
+				Error_2: Stack value must be integer, but met: {0}
+			}
+
+			ExampleSummonCommand: {
+				Usage:
+					'''
+					/summon type [[~]x] [[~]y] [number]
+					type - NPCID of NPC.
+					x and y - position of spawn.
+					~ - to use position relative to player.
+					number - number of NPC's to spawn.
+					'''
+				Description: Spawn a NPC by NPCID
+				Error_0: At least one argument was expected.
+				Error_1: "{0} is not a correct integer value."
+				Error_2: "{0} is not a correct X position (must be valid integer value)."
+				Error_3: "{0} is not a correct Y position (must be valid integer value)."
+				Error_4: "{0} is not a correct number (must be valid integer value)."
+			}
+
+			ExampleTimeCommand: {
+				Usage:
+					'''
+					/addTime numTicks
+					numTicks — positive or negative number in ticks (1 second = 60 ticks).
+					'''
+				Description: Adds numTicks to fast forward or rewind world time
+				Error_0: At least one argument was expected.
+				Error_1: "{0} is not a correct integer value."
+			}
```

Comment:
Recommend deduplicating some of these error messages. A simple `"{0} is not a valid number"` especially since the code doesn't actually check world bounds or any such. Maybe put them under `Commands.ErrorMessages.XXX`